src/routes/safe/store/actions/fetchTransactions.js

```diff
@@ -262,46 +260,88 @@ export type SafeTransactionsType = {
   cancel: Map<string, List<TransactionProps>>,
 }
 
+let cachedSafeTransactions = null
```

Comment:
we have latest transactions in the redux store, why do we need these variables? just return early and don't process transactions

Reply:
This is solved